src/components/PlayerInfo.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div :class="['player-info', player.faction]" v-if="player && player.faction" :style="`background-color: ${factionColor}`">
     <div class="text">
-      <b>{{name}}</b> - <span v-b-tooltip.hoover.html="tooltip" >{{faction}}</span> - {{data.victoryPoints}}vp <span v-if="passed">(passed)</span><br/>
+      <b>{{name}}</b> - <span v-b-modal="faction" >{{faction}}</span> - {{data.victoryPoints}}vp <span v-if="passed">(passed)</span>
+      <b-modal :id="faction" :title="faction">
```

Comment:
You should probably add `size='lg'`. Also, Move it outside (after `<div class='tiles'>`). Currently, if there are boosters / special actions / tech tiles, they are displayed over the modal.

Reply:
![image](https://user-images.githubusercontent.com/342922/43676757-76e369b8-97f7-11e8-8de7-f4fb46a6679f.png)

It will also solve the spacing after the first line of PlayerInfo. (another way to solve it would be to remove `<br>` or add `lazy="true"` to the modal)